chapel.html.markdown

```diff
@@ -683,34 +725,37 @@ for (i, j) in zip(toThisArray.domain, -100..#5) {
 }
 writeln(toThisArray);
 
-// This is very important in understanding why this statement exhibits a
-// runtime error.
+// This is very important in understanding why this statement exhibits a runtime error.
 
-/*
-  var iterArray : [1..10] int = [i in 1..10] if (i % 2 == 1) then i;
+/* 
+      var iterArray : [1..10] int = [i in 1..10] if (i % 2 == 1) then i;
 */
 
 // Even though the domain of the array and the loop-expression are
 // the same size, the body of the expression can be thought of as an iterator.
 // Because iterators can yield nothing, that iterator yields a different number
 // of things than the domain of the array or loop, which is not allowed.
 
-// Classes
-
+/*
+Classes
+*/
 // Classes are similar to those in C++ and Java, allocated on the heap.
 class MyClass {
 
 // Member variables
   var memberInt : int;
   var memberBool : bool = true;
 
-// Explicitly defined initializer.
-// We also get the compiler-generated initializer, with one argument per field.
-// Note that soon there will be no compiler-generated initializer when we
-// define any initializer(s) explicitly.
-  proc init(val : real) {
-    this.memberInt = ceil(val): int;
-  }
+// By default, any class that doesn't define an initializer gets a
+// compiler-generated initializer, with one argument per field and
+// the field's initial value as the argument's default value.
+// Alternatively, the user can define initializers manually as shown
+// in the following commented-out routine:
+//
+/*       // proc init(val : real) {
```

Comment:
Not sure why this is commented out, especially since the default initializer and this explicit initializer are not semantically equivalent.